app/views/restaurants/index.html.haml

```diff
@@ -3,3 +3,6 @@
 - @categories.each do |category|
   %h5= category.name
   %p= category.description
+- @restaurants.each do |restaurant|
+  = link_to restaurant.name, restaurant_path(restaurant)
+  %p= restaurant.address
```

Comment:
I would suggest displaying restaurants in their categories

Reply:
That's a good input, but it's not for this user story :)

Reply:
😎